src/utils/nodeTypeProcessing.ts

```diff
@@ -73,7 +72,9 @@ export function getParameterTypes(functionCall: FunctionCall, ast: AST): TypeNod
       );
       const structDef = functionType.type.to.definition;
       assert(structDef instanceof StructDefinition);
-      return structDef.vMembers.map(variableDeclarationToTypeNode);
+      return structDef.vMembers.map(
+        ast.inference.variableDeclarationToTypeNode.bind(ast.inference),
+      );
```

Comment:
Why does  the function needs to be bound to `ast.inference`

Reply:
I've changed that to use map's second argument instead